ambry-cloud/src/main/java/com/github/ambry/cloud/azure/AzureMetrics.java

```diff
@@ -142,6 +147,8 @@ public AzureMetrics(MetricRegistry registry) {
         registry.counter(MetricRegistry.name(AzureCloudDestination.class, BLOB_UPLOAD_REQUEST_COUNT));
     blobUploadSuccessCount =
         registry.counter(MetricRegistry.name(AzureCloudDestination.class, BLOB_UPLOAD_SUCCESS_COUNT));
+    blobUploadSuccessRate = registry.meter(MetricRegistry.name(AzureCloudDestination.class, BLOB_UPLOAD_SUCCESS_RATE));
+    blobUploadErrorCount = registry.counter(MetricRegistry.name(AzureCloudDestination.class, BLOB_UPLOAD_ERROR_COUNT));
```

Comment:
Are you planning to change the names to correct class (`CloudBlobStoreV2.class, etc`) later?

Reply:
Good catch !!